### ISSUE TITLE
Taskbar background invisible :(

### ISSUE THREAD

**Issue by Slint7** [Armin2208/Windows-Auto-Night-Mode#151]
**Describe the bug**
When I don't have an app running in full screen, the taskbar background is completely invisible.

**Expected behavior**
If I have an app in full screen then the taskbar background is semi-transparent like it should be.
If I don't have an app in full screen, then the taskbar background is missing or fully transparent, which is a problem.
This happened the moment I installed this app and then played around with the settings. Nothing else on my pc changed.

**Environment:**
 - Auto Dark Mode version number: 3.0.1
 - OS Version: Windows 10
 - OS language: English-US
 - Theme switching mode: Windows Theme
 - Has user admin rights?: Yes
 - AntiVirus software: Bitdefender
 - Similair apps: StartIsBack

**Settings**
I have the setting that windows is always in dark mode.

![Untitled-1](https://user-images.githubusercontent.com/69123622/89132379-f9302b80-d513-11ea-8393-afd39a2cbbb3.jpg)
![Untitled-2](https://user-images.githubusercontent.com/69123622/89132389-077e4780-d514-11ea-88f5-6fb94ead845a.jpg)
![Untitled-3](https://user-images.githubusercontent.com/69123622/89132449-8b383400-d514-11ea-9773-7d69b00b7bfd.jpg)

**Comment by Armin2208** [comment]
Looks like a typical, not replicable Windows 10 bug to me.
Does this even occurr after system restart?

**Comment by Slint7** [comment]
Hi. Thanks for the reply.

A system restart does not work, unfortunately.

I tried reinstalling the app and uninstall any other app that I thought
might clash with yours.

I also don't know how to properly uninstall your app without leaving
traces of the app to see if that fixes it?

Does your app change windows stuff? Must I try and fix it through regedit
or something?

Regards,
Josiah

**Comment by Armin2208** [comment]
The uninstaller already clears everything out (this is why you lose every user setting).
Auto Dark Mode only changes Registry Entries from Windows 10 itself.
You are using Windows Theme-File switching. How's normal Auto Dark Mode switching working, without a theme file?

**Comment by Slint7** [comment]
Hi

Using it without themes did not fix my problem, unfortunately, but did
change the colours of everything as it should.

Regards,
Josiah

**Comment by Armin2208** [comment]
oh, this issue seems more complex.
If you switc your theme in Windows Settings, does it change properly?

**Comment by emberaholdon** [comment]
Never had this problem, must be GPU/driver specific bug.

**Comment by Slint7** [comment]
Switching thems from Windows control panel works the same as the ADM app

**Comment by Slint7** [comment]
I have an RTX 2060 with latest drivers.

**Comment by Slint7** [comment]
It could be compatibility issues with another app on my pc, StartIsBack
which modifies inter alia the start menu and taskbar.

It could also be an update on another program or windows that I did not
think of.

**Comment by Armin2208** [comment]
seems like. weird!

**Comment by Slint7** [comment]
Hi

I found the problem and fixed it.

I just checked if any suspect apps updated around the time that I installed
Auto Night Mode and found that StartIsBack did. I then went to their
support forums and found their latest update broke stuff.

I thought it was because of Auto Night Mode since it was the only new app
on my pc.

I rolled back StartIsBack to a previous version and disabled auto-update.

My pc is working fine now.

Sorry for wasting your time.

I am impressed with your customer support. Thanks.

Regards,
Josiah
